### PR TITLE
standardize the keywords in the `get_*_metric` methods

### PR DESCRIPTION
**Pull request by bhazelton**
The keywords in these methods behaved very differently than the other get methods in mc_session.py, this PR standardizes them.

### CODE REVIEW

File: hera_mc/mc_session.py [hunk 1]
```diff
@@ -4764,25 +4856,35 @@ def update_metric_desc(self, metric, desc):
...
         -------
         list of MetricList objects
 
         """
-        args = []
+        query = self.query(MetricList)
+
         if metric is not None:
-            args.append(MetricList.metric.in_(get_iterable(metric)))
+            query = query.filter(MetricList.metric == metric)
```
Comment (mkolopanis) [inline hera_mc/mc_session.py:4882]: Any particular reason this is changed to only allow a single metric return? I feel like this multiple entry method _could_ be useful, though I bet it was hardly used.

**Review by mkolopanis — COMMENTED** [review]
This looks pretty good, I do have one small quesiton.

File: hera_mc/mc_session.py (hunk 1 above)
Comment (bhazelton) [reply hera_mc/mc_session.py:4882]: This was done in the name of unifying all the `get` methods on MCSession. We talked about it in a telecon and decided to make the metric get methods behave the same as other methods which do not accept lists only single filter values for each column.

Comment (mkolopanis) [reply hera_mc/mc_session.py:4882]: Ah fair enough, I must have missed that conversation.!

**Review by mkolopanis — APPROVED** [review]

**Pull request merged**